use-cases/run_jcl/README.md

```diff
@@ -0,0 +1,80 @@
+# Zowe CLI SonarQube Sample
```

Comment:
Please update header for the Run JCL sample

Reply:
Fixed